distributed/worker_memory.py

```diff
@@ -25,10 +25,17 @@
 import os
 import sys
 import warnings
-from collections.abc import Callable, MutableMapping
 from contextlib import suppress
 from functools import partial
-from typing import TYPE_CHECKING, Any, Container, Literal, cast
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Container,
+    Literal,
+    MutableMapping,
+    cast,
+)
```

Comment:
Please import objects from collections.abc instead of typing whenever possible.

Reply:
Not possible here afaict because they are not getitemable in py3.8 and I use them in a cast expression.

Reply:
You can use quotes `cast("Callable...", data`

Reply:
Done.